src/atoms/gv-input.js

```diff
@@ -298,11 +298,13 @@ export class GvInput extends InputElement(LitElement) {
 
       setTimeout(() => {
         const clipboardPopover = this.shadowRoot.querySelector('.clipboard__popover');
-        this.getInputElement().addEventListener('click', (e) => {
-          e.preventDefault();
-          e.stopPropagation();
-          clipboardPopover.click();
-        });
+        if (this.readonly) {
```

Comment:
don't know if it is related but I think there is a glitch on readonly story -> the value filled are not in the good format so the controls appears as invalid

![image](https://user-images.githubusercontent.com/47851994/106448090-93047100-6482-11eb-80f9-72d5bc46c875.png)

Reply:
Well seen, it was a mistake... 
A readonly field can't be invalid even with the wrong value

Reply:
✅